instaloader/__main__.py

```diff
@@ -20,7 +20,7 @@ def usage_string():
     return """
 {0} [--comments] [--geotags] [--stories] [--highlights] [--tagged]
 {2:{1}} [--login YOUR-USERNAME] [--fast-update]
-{2:{1}} profile | "#hashtag" | :stories | :feed | :saved
+{2:{1}} profile | "#hashtag" | %location_id | :stories | :feed | :saved
```

Comment:
Use

```suggestion
{2:{1}} profile | "#hashtag" | %%location_id | :stories | :feed | :saved
```

otherwise, `instaloader --help` fails with
```
TypeError: %o format: an integer is required, not dict
```

**Also**, please add a `g_targets.add_argument(...)` call in `__main__.py` line 214 to describe the new target in `--help` output.